### PR TITLE
Update installation.markdown

### PR DESCRIPTION
**Pull request by gozasc**
One grammatical change ("atleast" should be two separate words) and spacing/punctuation changes for the sake of consistency on the page.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -11,8 +11,8 @@ The following will take you through the steps required to install Home Assistant
 
      - [Raspberry Pi 3 Model B and B+ 32-bit][pi3-32] (recommended)
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 32-bit][pi4-32] (recommended)
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 64-bit][pi4-64]
```
Comment (frenck) [inline source/hassio/installation.markdown]: Please revert the change of removing the space (for all occurrences in this PR).

There should be a space between the value and its unit of measurement.

For reasoning and more information, see both the Microsoft & Google Style Guides on writing documentation.

- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms>
- <https://developers.google.com/style/units-of-measure>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/hassio/installation.markdown (hunk 1 above)
Comment (gozasc) [reply source/hassio/installation.markdown]: Sounds good. Thank you for letting me know about the Google/Microsoft guidelines regarding units of measure. 

If that is the case, please be sure to correct (by adding) a space to the bottom portion regarding VMware Workstation. There is a "2GB RAM" down there, which is what prompted me to recommend standardizing the spaces to begin with.

Comment (frenck) [reply source/hassio/installation.markdown]: Feel free to correct that as you process the review comments on your PR.

**Review by gozasc — COMMENTED** [review]
Resubmitting with added spaces between quantities and units of measurement (e.g. "2 GB") per the Google/Microsoft technical writing guidelines. Also added a space between "atleast" to make it two words as it should be.

File: source/hassio/installation.markdown [hunk 2]
```diff
@@ -12,7 +12,7 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 3 Model B and B+ 32-bit][pi3-32] (recommended)
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
      - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
+     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4GB model) 64-bit][pi4-64]
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
```

File: source/hassio/installation.markdown [hunk 3]
```diff
@@ -33,10 +33,10 @@ The following will take you through the steps required to install Home Assistant
...
    - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
-     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it atleast 2 GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter
-     - For Hyper-V create a new virtual machine, select "Generation 2", assign it at least 2 GB of memory and select "Connection -> "Your Virtual Switch that is bridged", then "Use an existing virtual hard disk" and select the VHDX file from above, after creation go to "Settings" -> "Security" and deselect "Enable Secure Boot"
+     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it at least 2GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter.
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it at least 2 GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterward edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @gozasc 👍

**Pull request merged**